app/models/member.rb

```diff
@@ -235,32 +229,31 @@ def update_newsletter_subscription
     end
   end
 
-  def newsletter_subscribe(testing=false)
-    return true if (Rails.env.test? && !testing)
+  def newsletter_subscribe(testing = false)
+    return true if Rails.env.test? && !testing
     gb = Gibbon::API.new
-    res = gb.lists.subscribe({
-      id: Growstuff::Application.config.newsletter_list_id,
-      email: { email: email },
-      merge_vars: { login_name: login_name },
-      double_optin: false # they already confirmed their email with us
-    })
+    gb.lists.subscribe({
+                          id: Growstuff::Application.config.newsletter_list_id,
```

Comment:
Not a huge fan of this style of indentation, as it runs counter to the rationale behind line length guidance (all roughly readable without too much scrolling as opposed to fits on a punchcard) and is pretty readable in the former format. If it's a specfic check as opposed to generic whitespace checks; might be worth turning it off.